library/std/src/sys/unix/stack_overflow.rs

```diff
@@ -143,11 +143,17 @@ mod imp {
     }
 
     unsafe fn get_stackp() -> *mut libc::c_void {
+        #[cfg(target_os = "openbsd")]
+        let flags = MAP_PRIVATE | MAP_ANON | libc::MAP_STACK;
+        // OpenBSD requires this flag for stack mapping
```

Comment:
"this flag" is referring to `MAP_STACK`? If so the comment is on the wrong line.

Reply:
indeed :)